packages/manager/apps/dedicated/client/app/billing/autoRenew/ssh/user-ssh.service.js

```diff
@@ -7,180 +7,181 @@ import set from 'lodash/set';
-        return key;
-      });
-    };
+  self.getDedicatedSshList = function getDedicatedSshList() {
+    return OvhHttp.get('/me/sshKey', {
+      rootPath: 'apiv6',
+    }).then((keyNames) => {
+      const promises = map(keyNames, (keyName) =>
+        self.getDedicatedSshKey(keyName),
+      );
+      return $q.allSettled(promises);
+    });
+  };
 
-    self.getCloudProjects = function getCloudProjects() {
-      return OvhHttp.get('/cloud/project', {
-        rootPath: 'apiv6',
-        cache: CLOUD_CACHE_KEY,
-        clearCache: false,
-      })
-        .then((projectIds) =>
-          $q.all(
-            map(projectIds, (projectId) =>
-              OvhHttp.get('/cloud/project/{serviceName}', {
-                rootPath: 'apiv6',
-                urlParams: {
-                  serviceName: projectId,
-                },
-                cache: CLOUD_CACHE_KEY,
-                clearCache: false,
-              }),
-            ),
-          ),
-        )
-        .then((projects) =>
-          map(filter(projects, { status: 'ok' }), (project) => ({
-            id: project.project_id,
-            description: project.description,
-          })),
-        );
-    };
+  self.getDedicatedSshKey = function getDedicatedSshKey(keyName) {
+    return OvhHttp.get('/me/sshKey/{keyName}', {
+      rootPath: 'apiv6',
+      urlParams: {
+        keyName,
+      },
+    }).then((key) => {
+      set(key, 'category', 'dedicated');
+      return key;
+    });
+  };
 
-    self.getCloudSshList = function getCloudSshList() {
-      return self
-        .getCloudProjects()
-        .then((projects) => {
-          const promises = map(projects, (project) =>
-            OvhHttp.get('/cloud/project/{serviceName}/sshkey', {
+  self.getCloudProjects = function getCloudProjects() {
+    return OvhHttp.get('/cloud/project', {
+      rootPath: 'apiv6',
+      cache: CLOUD_CACHE_KEY,
+      clearCache: false,
+    })
+      .then((projectIds) =>
+        $q.all(
+          map(projectIds, (projectId) =>
+            OvhHttp.get('/cloud/project/{serviceName}', {
               rootPath: 'apiv6',
               urlParams: {
-                serviceName: project.id,
+                serviceName: projectId,
               },
-            }).then((keys) =>
-              map(keys, (key) => ({
-                serviceName: project.id,
-                serviceDescription: project.description,
-                category: 'cloud',
-                keyName: key.name,
-                key: key.publicKey,
-                id: key.id,
-                url: self.getSshCloudUrl(project.id),
-              })),
-            ),
-          );
-          return $q.allSettled(promises);
-        })
-        .then((keysByProjet) => flatten(keysByProjet));
-    };
+              cache: CLOUD_CACHE_KEY,
+              clearCache: false,
+            }),
+          ),
+        ),
+      )
+      .then((projects) =>
+        map(filter(projects, { status: 'ok' }), (project) => ({
```

Comment:
`map` + `filter` can be `reduce` no ?

Reply:
the scope of this PR is to make billing section to be lazy loaded, so i'm trying to keep the original code untouched
(it appears as changed but only the indentation have)
i'm not planning any code refactoring here to not break anything